mkt/extensions/models.py

```diff
@@ -145,6 +145,13 @@ def get_fallback(cls):
     def get_indexer(self):
         return ExtensionIndexer
 
+    def is_dummy_content_for_qa(self):
+        """
+        Returns whether this extension is a dummy extension used for testing
```

Comment:
This is peculiar. Maybe add a note indicating why this is needed, rather than just what it does?